uint8 OFF = 0
uint8 HOVER = 1
uint8 RACING = 2
uint8 data
